styles.css

```diff
@@ -107,7 +188,43 @@ p {
   display: flex;
   position: relative;
   padding: 20px;
-  padding-bottom: 32%;
+  padding-bottom: 46.8%;
+}
+
+#title_section {
+  padding: none;
+  padding-top: 10%;
+}
+
+@media screen and (min-width: 800px) {
+  .section {
+    background-image: url(/images/Group\ 2.png);
+    background-position: right;
+    background-repeat: no-repeat;
+    height: calc(95vh - 150px);
+  }
```

Comment:
- [ ] As mentioned by the previous reviewer, this part of your project has not been properly fixed. There's a white space to the right of the page that has caused distortion to your page. At view port  `808px` has mentioned by the previous reviewer, this part of your project does not tally with what we have on the guiding template. Your main title overlapped the background image of the section. Your page should be responsive to the desktop sizes right from `768px` and above. Kindly look into that and fix please.

### Your Design
![your-headline](https://user-images.githubusercontent.com/26597432/207272522-715c9e0d-58d9-4bc2-866e-fed2d139ef3e.png)


[your-page-1.webm](https://user-images.githubusercontent.com/26597432/207273138-610f2666-c8b2-4a83-9646-43c51b043f4e.webm)


### Figma Desing
![figma-headline](https://user-images.githubusercontent.com/26597432/207272568-c9107708-e608-451d-93d2-bd5672a780c2.png)